Hotsapi.Parser/Program.cs

```diff
@@ -25,12 +28,13 @@ public static int Main(string[] args)
                     Console.Error.WriteLine($"File '{args[0]}' does not exist");
                     return 1;
                 }
-                var result = DataParser.ParseReplay(args[0], false, false, skipUnitParsing: true, skipMouseMoveEvents: true);
-                if (result.Item1 != DataParser.ReplayParseResult.Success || result.Item2 == null) {
-                    Console.Error.WriteLine($"Error parsing replay: {result.Item1}");
+
+                var result = ParseReplay(args[0]);
+                if (result == null)
+                {
```

Comment:
We either need to switch away from egyptian brackets or use them everywhere, mixing styles is a bad idea